GoRest.Api.Client/Client/Builders/Comments/CreateCommentBuilder.cs

```diff
@@ -0,0 +1,37 @@
+﻿using GoRest.Api.Client.Client.Models;
```

Comment:
Rename pool request. As I know it includes todos as well

Reply:
Done